### ISSUE TITLE
[Convention Proposal] Unknown Dupe BCME

### ISSUE THREAD

**Issue by pianoblook** [hanabi/hanabi.github.io#979]
As presented in the broader Dupe Tech [doc](https://docs.google.com/document/d/1zhFHjb2ChoLEIKn71bd8yKFczy0kn3hStrydRQ_m4NE/edit?usp=sharing), this is a cute + powerful hybrid combining the concepts of _Bad Chop Moves_ + _Unknown Trash_.

As written in the doc:
### Unknown Dupe Bad Chop Move Ejection
- First, see the section on [Bad Chop Move Ejections](https://hanabi.github.io/docs/extras/ejections/#the-bad-chop-move-ejection).
- If a _Chop Move_ is used to _CM_ nothing useful except for _Unknown Duplicates_, this is very odd.  Bob must signal the “badness” of the cards by _Ejecting_.
- Cathy will mark both cards as _Known Trash._ After eventually discarding one of them, she will mark the remaining card as exactly the duplicate of the discarded cards.
- Excitingly, Cathy has the ability to _TOCM_ with these dupes, since she was just told they were all Trash!

**Examples:**

- In game: https://hanab.live/replay/791290#40
- In a HuLi Hypo: [Turn 6](https://hanab.live/shared-replay-json/315iccfbionbghfafkku-werlpnxldwtsapaugxhr-vqyvsjmkmpuqd,03pbxa-antclaaokbgaaqbdajak-bbsaambebfobavaixaac-gatbazay1aaslablaab3-bpb4lcb5b6wawbaBa0sa-aCat1baxaDtbocaHawa2-ag1aaGbhbua1,0) 

^ Note both those examples also use the _Unnecessary Trash Assumption_, though the general idea remains the same even without access to that.

**Comment by T-Hoodie** [comment]
What happens if someone chop move 3 cards, including a duplicated pair?

**Comment by pianoblook** [comment]
> What happens if someone chop move 3 cards, including a duplicated pair?

If the 3rd card isn't trash, then it wouldn't count as a BCME. 

If extra trash is CM'd, then you can still BCME (see the in-game example)

**Comment by pianoblook** [comment]
I definitely would disagree that it wouldn't be helpful to define. You could certainly make arguments for a few responses.

Here are the main ones I'd consider if I saw this move in the wild: 
- no response necessary (maybe Alice was just looking for an excuse to stall?)
- trash pull layered finesse (since it wasn't possible to good touch one copy)
- trash pull discharge (maybe my slot 3 is tricky to get, with UDD either being blocked or unpreferable)
- BCME, ofc.

So I think there's a strong case for BCM interpretation to take precedence and hence that it should be officially codified, as to reduce error.

**THAT SAID**, I might have been convinced to *disallow* this in cases where 3 cards get cm'd, since committing to a BCME interpretation in that scenario loses a significant amount of its utility.

**Comment by Zamiell** [comment]
blocked by #947

**Comment by vEnhance** [comment]
Dupe tech is merged now, so unblocking this

**Comment by Zamiell** [comment]
im hesitant to add any more rarely-used moves to the doc
